_schema.graphql

```diff
@@ -1303,6 +1303,7 @@ type Submission {
   attributionClass: AttributionClass
   authenticityCertificate: Boolean
   category: String
+  consignmentState: String
```

Comment:
**question/suggestion**: after removing the references to the word consignment, I think we are trying to avoid using it in our codebase as much as possible. do you think we can rename this to `submissionState` or `submissionStatus`?

Reply:
Updated to `sale_state`